string type
geometry_msgs/Pose2D geometry
